src/lst/LST.jsx

```diff
@@ -1,128 +1,101 @@
 import LSTDistributionGraph from './LSTDistributionGraph';
+import LSTList from './LSTList';
+import LSTTotalValue from './LSTTotalValue';
 import { ParentSize } from '@visx/responsive';
+import { Spinner } from '@nextui-org/react';
+import { useEffect } from 'react';
+import { useMutativeReducer } from 'use-mutative';
+import { useServices } from '../@services/ServiceContext';
 
 export default function LST() {
   const { eigenlayerService } = useServices();
   const [state, dispatch] = useMutativeReducer(reduceState, {
+    error: undefined,
+    ethRate: 1,
     isLoadingLST: true,
-    rankings: [],
     lst: [],
-    latestRate: 1,
-    tvl: 0,
-    percentage: 0
+    rankings: []
   });
 
-  useEffect(() => {
-    async function fetchEigenLayerTVL() {
-      const data = await eigenlayerService.getEigenLayerTVLOvertime();
-      if (data.length > 0) {
-        const current = data[data.length - 1];
-
-        dispatch({
-          tvl: parseFloat(formatEther(current.lstTVL)),
-          percentage:
-            (parseFloat(current.lstTVL) /
-              (parseFloat(current.lstTVL) + parseFloat(current.ethTVL))) *
-            100
-        });
-      }
-    }
-
-    fetchEigenLayerTVL();
-  }, []);
-
   useEffect(() => {
     async function fetchLSTTotalValue() {
-      dispatch({ isLoadingLST: true });
-      const data = await eigenlayerService.getEigenLayerLSTTotalValue();
-      if (data.length > 0) {
-        const current = data[data.length - 1];
-        const rankings = Object.entries(current.tvl);
-        rankings.sort((a, b) => {
-          if (a[1] < b[1]) {
-            return 1;
-          }
+      dispatch({ isLoadingLST: true, error: undefined });
+      try {
+        const data = await eigenlayerService.getEigenLayerLSTTotalValue();
+        if (data.length > 0) {
+          const current = data[data.length - 1];
+          const rankings = Object.entries(current.tvl);
+          rankings.sort((a, b) => {
+            if (a[1] < b[1]) {
+              return 1;
+            }
 
-          if (a[1] > b[1]) {
-            return -1;
-          }
+            if (a[1] > b[1]) {
+              return -1;
+            }
 
-          return 0;
-        });
+            return 0;
+          });
 
+          dispatch({
+            rankings,
+            ethRate: current.rate,
+            lst: data,
+            isLoadingLST: false
+          });
+        }
+      } catch (e) {
         dispatch({
-          rankings,
-          latestRate: current.rate,
-          lst: data,
-          isLoadingLST: false
+          isLoadingLST: false,
+          error: handleServiceError(e)
         });
       }
     }
 
     fetchLSTTotalValue();
-  }, []);
+  }, [dispatch, eigenlayerService]);
 
   return (
     <div>
-      <div className="my-4 font-display text-3xl font-medium text-foreground-1">
+      <div className="mb-4 font-display text-3xl font-medium text-foreground-1">
         Liquid Staking Tokens
```

Comment:
Capitalization must be like everywhere else:

```suggestion
        Liquid staking tokens
```